quinn-h3/src/server.rs

```diff
@@ -28,6 +135,7 @@ use crate::{
     try_take, Error, Settings,
 };
 
+/// Configure and build a HTTP/3.0 server.
```

Comment:
General comment about most of the docstrings: in quinn/quinn-proto we've chosen to not end most first lines of docstrings with "." on the idea that these are usually sentence fragments, not full sentences.